phc/easy/query/api_paging.py

```diff
@@ -0,0 +1,121 @@
+import json
+import pandas as pd
+from urllib.parse import urljoin
+from phc.base_client import BaseClient
+from typing import List, Union, Optional
+from phc.easy.auth import Auth
+from phc.easy.util import tqdm
+
+MAX_RESULT_SIZE = 999
+
+
+def clean_params(params: dict):
+    return {
+        k: v
+        for k, v in params.items()
+        if ((v is not None) and (not isinstance(v, str) or len(v) > 0))
+    }
+
+
+def recursive_paging_api_call(
```

Comment:
We want this to be somewhat parallel to the [recursive_execute_fhir_dsl(...) function](https://github.com/lifeomic/phc-sdk-py/blob/70e4da931b02e130952a3005245f4a13796f44ab/phc/easy/query/fhir_dsl.py#L82-L141).